data/core/tips.json

```diff
@@ -202,23 +197,18 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Like to gamble?  Use mutagen!  Be a winner today!" ]
+    "text": [ "Mutagens can have unpredictable and dangerous effects.  You're gambling to get their advantages." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Smoking and napping is asking for more than a hole in your shirt." ]
+    "text": [ "Lit cigarettes can start fires. Falling asleep smoking isn't recommended." ]
```

Comment:
```suggestion
    "text": [ "Lit cigarettes can start fires.  Falling asleep smoking isn't recommended." ]
```

Reply:
This sounds like FDA approved message. What was wrong with previous one?